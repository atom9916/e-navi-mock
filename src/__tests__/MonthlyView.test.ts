/**
 * @jest-environment jsdom
 */
import { render, screen } from '@testing-library/vue'
import MonthlyViewVue from '@/views/MonthlyView.vue'

describe('MonthlyViewビューのテスト', () => {
  test('画面が正しく表示されている', () => {
    render(MonthlyViewVue)

    const h2 = screen.getByRole('heading')

    expect(h2).toHaveTextContent('月次勤怠')
  })
  test('スナップショットテスト', () => {
    const { container } = render(MonthlyViewVue)

    expect(container).toMatchSnapshot()
  })
})
