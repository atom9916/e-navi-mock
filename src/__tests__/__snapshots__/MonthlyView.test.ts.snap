// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MonthlyViewビューのテスト スナップショットテスト 1`] = `
<div>
  <h2>
    月次勤怠
  </h2>
</div>
`;
